features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Environments/Environments.jsx

```diff
@@ -291,8 +290,7 @@ const useStyles = makeStyles((theme) => ({
  */
 export default function Environments() {
     const classes = useStyles();
-    const theme = useTheme();
-    const { maxCommentLength } = theme.custom;
+    const maxCommentLength = '255';
```

Comment:
Why do we need to hardcode this ? can't we get this com theme or settings file , and ideally I think this should read from OAS definition

Reply:
This validation happens from the backend. that constant is used here